### PR TITLE
Current state of prod

### PR DESCRIPTION
**Pull request by liam-middlebrook**
* Add a config for the haddock dictionary location
* Update default packet start-date for new entires
* Add check to ignore failed-onfloor members from the upperclassmen listing

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@
     t.boolean  "doing_packet"
     t.boolean  "on_packet"
     t.string   "announcement"
-    t.date     "start_date",         default: '2016-08-21'
+    t.date     "start_date",         default: '2017-08-27'
```
Comment (TalCohen) [inline db/schema.rb]: This seems odd to me that it was ever like this. It would be better to just set the default date in the model when creating the record.

I'm okay with this for now but ideally we would fix this for future us to not have to always change this.

File: db/packet_ldap.rb [hunk 2]
```diff
@@ -5,6 +5,7 @@
 LDAP_EVAL_BASE   = 'cn=Evaulations,ou=Committees,dc=csh,dc=rit,dc=edu'.freeze
 LDAP_EBOARD_CN   = 'eboard'.freeze
 LDAP_INTRO_CN    = 'intromembers'.freeze 
+LDAP_FAILED_CN    = 'failed'.freeze
```
Comment (TalCohen) [inline db/packet_ldap.rb]: What's an example for when this is necessary?

File: config/environment.rb [hunk 3]
```diff
@@ -3,3 +3,6 @@
 
 # Initialize the Rails application.
 Packet::Application.initialize!
+
+# Haddock Dictionary for Password Generation
+Haddock::Password.diction = "words.txt"
```
Comment (TalCohen) [inline config/environment.rb]: Why was this change needed? Where is words.txt?

File: db/packet_ldap.rb (hunk 2 above)
Comment (liam-middlebrook) [reply db/packet_ldap.rb]: If someone who passed their Introductory Evaluations process gets onto the housing board. Then they proceed to fail Membership Evaluations. If they stay on the roster for living on floor the next year they will have a disabled account (with the roomNumber field updated) but would have no authority to participate in the packet process as an upperclassman.

File: config/environment.rb (hunk 3 above)
Comment (liam-middlebrook) [reply config/environment.rb]: This change was needed in order to get Haddock to properly generate passwords. Words.txt is downloaded separately into the repo's root directory.
